libs/core-components/src/lib/FieldWrapper/FieldWrapper.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
If we haven't needed to use the non-Formik `FieldWrapper` until now, let's maybe reconsider whether it's appropriate to use it extensively in the MFA flow. I identify 2 uses of this component that I think are actually pretty distinct: 
- Wrapping an individual field within a form, where we would use `FieldWrapperFormik` if Formik was being used to control the form.
- Wrapping a piece of UI that makes an API call, and handling possible errors. For instance the QR code generation UI is wrapped in a `FieldWrapper`, even though it has no inputs.

I suggest we migrate forms with actual inputs over to Formik, which should cover the first use case (I can put together a diff for this). If we do this, then I think it becomes clear that `FieldWrapper` isn't really a mechanism for wrapping _form fields_, it's a mechanism for wrapping _mutations_. We can think about what a `MutationWrapper` would look like, but for now it might make more sense to keep the error handling inline.

Reply:
I've removed `FieldWrapper`. I used it to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)-add class names[^1], but I see the problem with using its `error` prop.[^2] Designers wanted errors under fields (like field errors)[^3], so I added code to allow an `<InlineMessage>` to do so.

[^1]: To resolve this problem a different way, I added `.s-form` which automatically styles based on markup within. This class is not in Core-Styles yet. Until it is added, I support (in tup-ui global css) a minimal version of what it will be.

[^2]: Wes, you didn't understand _mutations_. He explained it. His concern was "using the 'error' attribute of FieldWrapper to display _network errors_ instead of _validation errors_." Valid concern. (You have previously referred to them as "form errors" and "field errors".) If a `FieldWrapper` error is used correctly, it is for a _validation error_, but if a field has validation errors, then it should be using `FieldWrapperFormik`.

[^3]: Because they didn't distinguish form errors and field errors on a one-field form. I may revisit this with them later.